package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@testing-library/jest-dom": "^4.2.0",
```

Comment:
Why were these added? they should already be included. Same with some of your jest stuff below

Reply:
it was part of the dependencies I installed. Oversight